coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -152,9 +168,30 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
             raise KeyError("Language {!r} is not defined for docstyle {!r}."
                            .format(language, docstyle))
 
+        def get_from_docstyle_settings(*args):
+            values = list()
+            for req_setting in args:
+                setting = dict(
+                    docstyle_settings.contents.items()).get(req_setting)
+                try:
+                    setting = setting.value
+                except AttributeError:
+                    setting = ""
+                values.append(setting)
+            return values
+
+        metadata_settings = ("param_start", "param_end", "return")
+
+        values = get_from_docstyle_settings('param_start', 'param_end',
+                                            'return_sep')
```

Comment:
`return_sep` does not match `return` from above

also you should use
```python
values = get_from_docstyle_settings(*metadata_settings)
```
to avoid duplication^^